app/controllers/scientific_names_controller.rb

```diff
@@ -25,23 +26,22 @@ def new
   end
 
   # GET /scientific_names/1/edit
-  def edit
-  end
+  def edit; end
 
   # POST /scientific_names
   # POST /scientific_names.json
   def create
     @scientific_name = ScientificName.new(scientific_name_params)
     @scientific_name.creator = current_member
 
-    flash[:notice] = 'Scientific name was successfully created.' if @scientific_name.save
+    @scientific_name.save
     respond_with(@scientific_name.crop)
   end
 
   # PUT /scientific_names/1
   # PUT /scientific_names/1.json
   def update
-    flash[:notice] = 'Scientific name was successfully updated.' if @scientific_name.update(scientific_name_params)
```

Comment:
the flash comes from the responders gem now. so we don't have to repeat ourselves AND it'll be translated already.